common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -26,6 +26,71 @@
     }
 }
 
+@if(isDemandTest) {
+    <div class="js-view-tracking-component submeta user__question">
+
+        <span class="js-storyquestion-atom-id is-hidden" id="@storyquestions.id"></span>
```

Comment:
You can get rid of that element, it is already defined below. In hindsight, it should have been a `data-` attribute instead, but that is none of your concern.

Reply:
Sorry I meant to add, _only if_ both HTML fragments appear at the same time. Looking at the tests, I'm not sure, hence me asking for screenshots. I should've be clearer 😅  What's bugging me is it is easy to have both `isDemandTest` and `!isAmp && !isClosed` both true or both false. Just wondering if this wouldn't make more sense:
```
if(!isAmp && !isClosed) {
  <-- Here we know we must display the atom -->
  if (isDemandTest) {
    <-- Here we display the "with delivery 404 test" variant -->
  } else {
    <-- Here we display the control variant -->
  }
}
```

Reply:
I don't believe there is a control variant here, which is why I had initially moved it out of the `!isAmp &&!isClosed` logic. We just need to display the test html for as long as the switch is on.

Reply:
So from checking the HTML of running this code locally, it doesn't appear that the HTML fragment in the `!isAmp &&!isClosed` branch is present. What do you think would be the best way to structure the control flow?

Reply:
Hmm, that is weird. Here are two scenarios I would test:

- `isAmp` is true
- `isClosed` is true

Currently, everything inside `if(isDemandTest){...}` will be displayed in both cases, which means in AMP pages and even when the storyquestions atom has been closed.

- `isAmp` and `isClosed` are both false

In this combination, you should see not only your own test, but also what I call the control variant, i.e. the original storquestions atom.